backends-velox/src/test/scala/org/apache/spark/sql/execution/VeloxParquetWriteForHiveSuite.scala

```diff
@@ -87,6 +87,21 @@ class VeloxParquetWriteForHiveSuite extends GlutenQueryTest with SQLTestUtils {
         _.getMessage.toString.contains("Use Gluten parquet write for hive")) == native)
   }
 
+  test("test hive static partition write table") {
+    withTable("t") {
+      spark.sql(
+        "CREATE TABLE t (c int, d long, e long)" +
+          " STORED AS PARQUET partitioned by (c, d)")
+      withSQLConf("spark.sql.hive.convertMetastoreParquet" -> "true") {
+        checkNativeWrite(
+          "INSERT OVERWRITE TABLE t partition(c=1, d=2)" +
+            " SELECT 3 as e",
+          native = false)
```

Comment:
```suggestion
          native = true)
```
should we set native = true to confirm the native write works?

Reply:
@zjuwangg I have updated the verify code. Please help to review again. Thanks.